File: CPP/Leet Code Solutions/two-sum/two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int [] answerIndex = new int[2];
        int c = 0;
        for(int i=0;i<nums.length;i++){
            for(int j=i+1;j<nums.length;j++){
                if(nums[i]+nums[j]==target){
                    answerIndex[c] = i;
                    c++;
                    answerIndex[c] = j;
                }
            }
        }
        return answerIndex;
    }
}

File: leetcode-questions/two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        map<int,int> m;
        vector<int> res;
        for(int i=0;i<nums.size();i++){
            if(m.find(target-nums[i])!=m.end()){
                res.push_back(i);
                res.push_back(m[target-nums[i]]);
                return res;
            }else {
                m[nums[i]]=i;    
            }
        }
        return res;
    }
};
